ur_robot_driver/resources/ursim_driver/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: "3.3"
+services:
+  driver:
+   image: driver_img
+   privileged: true
+   environment:
+    - DISPLAY=:1
+   networks:
+    static_test_net:
+     ipv4_address: 192.168.56.1
+   ports:
+      - "5566:5566"
+   devices:
+     - "/tmp/ttyUR:/tmp/ttyUR"
+   build:
+    context: driver/.
+    dockerfile: Dockerfile
+    args:
+    - ROS2_DISTRO=${ROS2_DISTRO}
+    - BUILD_MOVEIT2=${BUILD_MOVEIT2}
+   depends_on:
+      - "ursim"
+  ursim:
+   image: ursim_img
+   privileged: true
+   environment:
+    - ROBOT_MODEL=UR5
+    - DISPLAY=:1
+   networks:
+    static_test_net:
+     ipv4_address: 192.168.56.101
+   ports:
+      - "5900:5900"
+      - "29999:29999"
+      - "8080:8080"
+      - "30001-30004:30001-30004"
+   volumes:
+      - ./ursim/aux/.vol/:/ursim/programs
+   build:
+    context: ursim/.
+    dockerfile: g5/Dockerfile
```

Comment:
We could use the upstream docker images released by UR by now. But that's not a must.